StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetUpdateContext.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PaymentSheetUpdateContext.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 3/24/23.
+//
+
+import Foundation
+
+struct UpdateContext {
```

Comment:
IMO this should be in the FlowController file, it's not useful outside of it.  Either way we should nest it in the FlowController type.